apollo-openapi/src/main/java/com/ctrip/framework/apollo/openapi/client/url/OpenApiPathBuilder.java

```diff
@@ -84,7 +86,12 @@ public OpenApiPathBuilder namespacesPathVal(String namespaces) {
   }
 
   public OpenApiPathBuilder itemsPathVal(String items) {
-    pathVariable.put(ITEMS_PATH, escapePath(items));
+    if (UrlUtils.hasIllegalChar(items)) {
+      pathVariable.put(ENCODE_ITEMS_PATH, UrlUtils.encode(items));
```

Comment:
Is it necessary to use `escapePath` for the base64 encoded string?

Reply:
it's necessary. 
1. for example, if the item key is Chinese character or other special character(except /  \ .).  use escapePath may more suitable.
2. when openapi client is in high level, but portal is in low level. it can also work fine.